spec/models/group_spec.rb

```diff
@@ -41,6 +56,10 @@
     it 'returns the group\'s repository name' do
       expect(group.repository_name).to eq 'g2markus'
     end
+
+    # it 'rejects special chars repo_name' do
```

Comment:
I'm guessing this test should work now? Seems like the `update` call should return `false` instead of `true` though.